src/main/java/Duke.java

```diff
@@ -1,10 +1,35 @@
+import duke.*;
+
+import java.io.File;
+import java.io.IOException;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    public Duke(String path, String fileName) {
+        this.storage = new Storage(path,fileName);
+        this.tasks = storage.load();
+        this.ui = new Ui(this.tasks);
+    }
+
+    public void run() throws IOException, EmptyInputException, NoResponseException {
```

Comment:
Can consider using try catch to handle the exception and display the error message?